Providers/nxOMSAutomationWorker/automationworker/3.x/scripts/importHelper.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# ====================================
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+# ====================================
+
+PY_MAJOR_VERSION = 0
+PY_MINOR_VERSION = 2
```

Comment:
is this needed ?

Reply:
This is indeed required when python2 scripts are run in python3 hybrid worker environment